### PR TITLE
Changed indexfile#114

### PR DESCRIPTION
**Pull request by aditya-369**

### CODE REVIEW

File: assets/css/translation.css [hunk 1]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#navcontainerPosition {
```
Comment (shardulc) [inline assets/css/translation.css]: `navContainerPosition`

File: assets/css/translation.css [hunk 2]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#divPullRightHidden {
+    width: 35%;
+}
+
+#faFaGlobe {
```
Comment (shardulc) [inline assets/css/translation.css]: `faGlobe`, and `padding: 5px 0` (can this same style be used for other icons? If yes, it should be a class.)

File: assets/css/translation.css [hunk 3]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+
+#divForUploadError {
+    min-height: 50px;
+    margin-top: 1em;
+}
+
+#uploadError {
+    display: none;
+}
+.progress {
```
Comment (shardulc) [inline assets/css/translation.css]: Blank line between rules (everywhere :smiley:). Also, this is used only once?

File: assets/css/translation.css [hunk 4]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+
+#morphAnalyzerOutput {
+    border-spacing: 0;
+}
+
+#selectLanguage {
+    padding: 5px 0 0 5px;
+}
+
+#footer-container-unoderlist {
```
Comment (shardulc) [inline assets/css/translation.css]: camelCase, not hyphenated-words :smiley:

File: assets/css/translation.css [hunk 5]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+#divForMaintainer {
+    padding-bottom: 2em;
+}
+
+#refForGsoc,
+#refForCodeIn {
+    text-decoration: none;
+}
+
+#imgOfGsocLogo {
```
Comment (shardulc) [inline assets/css/translation.css]: These names are a little too verbose. Just `gsocLogo` would do?

File: index.html.in [hunk 6]
```diff
@@ -208,22 +208,22 @@
...
-                                          <div class="lead" id="fileName" style="display: none;"></div>
-                                          <div style="min-height: 50px; margin-top: 1em;">
-                                              <span id="uploadError" class="text-danger lead" style="display: none"></span>
-                                              <div class="progress progress-striped active" style="display: none; margin-bottom: 0px;">
-                                                  <div id="fileUploadProgress" class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" style="width: 0%"></div>
+                                          <div class="lead" id="fileName"></div>
+                                          <div id="divForUploadError">
+                                              <span id="uploadError" class="text-danger lead"></span>
+                                              <div class="progress progress-striped active" id="seventn">
```
Comment (shardulc) [inline index.html.in]: What does `seventn` do?

File: index.html.in [hunk 7]
```diff
@@ -87,13 +87,13 @@
                       <div>
                           <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" alt="Apertium Box" class="apertiumBox">
                           <span class="apertiumSubLogo"></span>
-                          <span class="apertiumLogo" style="margin-bottom: -.25em">Apertium</span>
+                          <span class="apertiumLogo" id = "apertiumLogo" >Apertium</span>
```
Comment (shardulc) [inline index.html.in]: Space at the end of the opening tag

File: assets/css/translation.css [hunk 8]
```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
...
+    margin-top: .5em;
+    width: 100%;
+}
+
+#imgOfSourceforge {
+    margin-top: .25em;
+    width: 100%;
+}
+
+#divForLicenses,
```
Comment (shardulc) [inline assets/css/translation.css]: These names are similarly verbose.

**Comment by sushain97** [comment]
I think this is worth mentioning again: there are a BUNCH of utility classes in Bootstrap 4: https://getbootstrap.com/docs/4.0/utilities (see left navigation for the full listing of topics for them). ANY new rules that are created in this PR which are subsets of those rules should instead be replaced with Bootstrap 4 utility classes that are created in `general.css` in anticipation of our migration to Bootstrap 4 when they can be deleted.

File: assets/css/translation.css (hunk 2 above)
Comment (aditya-369) [reply assets/css/translation.css]: I did not find same padding styles for other icons.Sorry if mistaken.

File: assets/css/translation.css (hunk 3 above)
Comment (aditya-369) [reply assets/css/translation.css]: ya had to use class as this was uneffected i.e, never progress class is used else where but here.

**Pull request closed without merging**